stratum/hal/lib/p4/p4_info_manager.h

```diff
@@ -293,6 +298,8 @@ class P4InfoManager {
   absl::flat_hash_map<uint32, std::string> id_to_resource_type_map_;
   google::protobuf::RepeatedPtrField<p4::config::v1::DirectPacketModMeter>
       direct_meter_objects;
+  google::protobuf::RepeatedPtrField<p4::config::v1::PacketModMeter>
+      meter_objects;
```

Comment:
```suggestion
      all_meter_objects_;
```
Member variable names end in an underscore. That's the convention.

Also, the other member variables that serve this purpose have an `all_` prefix. I'm afraid I missed this when I reviewed the direct meter code.

While you're at it, please make the same corrections for `direct_meter_objects`.

Reply:
Fixed